### PR TITLE
tests: add workload to realtikvtest/addindexlittest

### PR DESCRIPTION
**Pull request by tangenta**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: tests/realtikvtest/addindexlittest/workload.go [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+		if ctx.EndWorkload {
+			break
+		}
+		if w.loadType == 0 {
+			// insert workload
+			err = insertWorker(ctx, tk)
+		} else if w.loadType == 1 {
+			// update workload
+			err = updateWorker(ctx, tk)
+		} else if w.loadType == 1 {
```
Comment (Benjamin2037) [inline tests/realtikvtest/addindexlittest/workload.go:94]:
```suggestion
		} else if w.loadType == 2 {
```

File: ddl/index.go [hunk 2]
```diff
@@ -751,24 +756,34 @@ func goFastDDLBackfill(w *worker, d *ddlCtx, t *meta.Meta, job *model.Job,
...
 		} else {
+			if !IsEnableFastReorg() {
+				// Currently rollback add index job.
+				// With Checkpoint, if index is not backfill by lightning, then still could go txn-merge way to finish build up index.
+				ver, err = convertAddIdxJob2RollbackJob(d, t, job, tbl.Meta(), indexInfo, err)
+				if err1 := rh.RemoveDDLReorgHandle(job, reorgInfo.elements); err1 != nil {
+					logutil.BgLogger().Warn("[ddl] run add index job failed, convert job to rollback, RemoveDDLReorgHandle failed", zap.String("job", job.String()), zap.Error(err1))
+				}
+			}
 			// Be here, means the DDL Owner changed or restarted, the reorg state is re-entered.
```
Comment (Benjamin2037) [inline ddl/index.go:767]: This check empty table，can we put it to line before 728?

File: ddl/index.go [hunk 3]
```diff
@@ -751,24 +756,34 @@ func goFastDDLBackfill(w *worker, d *ddlCtx, t *meta.Meta, job *model.Job,
...
-				if err := rh.RemoveDDLReorgHandle(job, reorgInfo.elements); err != nil {
-					logutil.BgLogger().Warn("[ddl] Lightning: removeDDLReorgHandle failed", zap.Error(err))
-					return false, 0, errors.Trace(err)
-				}
-				job.SnapshotVer = 0
-				reorgInfo, err = getReorgInfo(d.jobContext(job), d, rh, job, tbl, elements)
-				if err != nil {
-					logutil.BgLogger().Warn("[ddl] Lightning: resumeDDLReorgHandle failed", zap.Error(err))
-					return false, 0, errors.Trace(err)
+			if lightning.GlobalEnv.IsInited {
```
Comment (Benjamin2037) [inline ddl/index.go:773]: Here, we still have a corner case, if we first enable and use lightning to do backfill, but after restart or change owner, the new DDL owner could not init env of lightning, maybe cause problem.

Comment (Benjamin2037) [reply ddl/index.go:773]: I thought without checkpoint or keep status in reorg record in TikV,  some case can hardly be handle easy and simple.

File: tests/realtikvtest/addindexlittest/workload.go (hunk 1 above)
Comment (tangenta) [reply tests/realtikvtest/addindexlittest/workload.go:95]: Let's keep it to reproduce the issue until we solve it.

**Pull request closed without merging**